_query-dsl/joining/parent-id.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: default
+title: Parent ID
+parent: Joining queries
+nav_order: 40
+---
+
+# Parent ID query
+
+The `parent_id` query returns child documents whose parent document has the specified ID. You can establish parent-child relationships between documents in the same index by using a [join]({{site.url}}{{site.baseurl}}/field-types/supported-field-types/join/) field type.
+
```

Comment:
Above, global: We use "parent/child" (forward slash) above but "parent-child" here. Should be consistent globally.